app/views/users/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="users container">
+  <ul class="user_list col">
+    <% @set_users.each do |user| %>
+      <li class="user_item row">
+        <div class="u_img">
+          <img src=<%= user.photo %> alt="u_img">
+        </div>
+        <div class="user_cont">
+          <a href=<%= user_url(user.id) %>>
+            <div class="user_desc col">
+              <h2 class="user_name"><%= user.name %></h2>
+              <p class="user_post_n">Number of posts: <%= user.posts_count %></p>  
+            </div>
```

Comment:
- [ ] Please use correct table column name for now you using `posts_count` but in database table column name is `post_counter` so kindly fix because its give an error check below the error for more clarification :slightly_smiling_face: 

![Screenshot from 2023-11-26 02-27-35](https://github.com/BilalLiaquat7/blog_app/assets/45226848/0653aaa2-1575-4785-893b-121035149524)